docs/i18n/fr/troubleshooting.md

```
@@ -16,3 +16,89 @@ Source anglaise:
+
+**Options de correction** :
+
+1. Passer à Brave :
+```toml
+[web_search]
+enabled = true
+provider = "brave"
+brave_api_key = "<SECRET>"
+```
+
+2. Passer à Exa :
```

markdownlint-cli2 (0.21.0)

[warning] Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)